### DISCUSSION TITLE
Add "Download" feature to Android version

### DISCUSSION THREAD

**Discussion by RafaelLinux** [zaps166/QMPlay2#675] [Ideas]
Hello
As it's not an issue, but a feature request, I put it here. Please, add "Download" option to "MediaBrowser" tab list in Android version, as in desktop one.

Thank you

**Comment by zaps166** [comment]
Android version already have it, but it was missing tap-and-hold for popup menu. Now it should be possible!

**Comment by RafaelLinux** [comment]
Thanks !!
